src/rustdoc/membership.md

```diff
@@ -0,0 +1,116 @@
+
+[CoC]: https://www.rust-lang.org/policies/code-of-conduct
+
+Being promoted to member implies a number of privileges:
+
+- Members have `r+` (approve a pull request) privileges and can do reviews (they are expected to
+  use those powers appropriately, as discussed previously). They also have access to control
+  perf/rustc-timer and other similar bots. See the documentation for `bors` and `r+`
+  [here](https://rustc-dev-guide.rust-lang.org/contributing.html#r-1).
+
+  Tip: some baseline rules around bors permissions are: don't do a `try` build unless you have
+  done a check for malicious code first and don't `r+` unless you are reasonably confident that
+  you can effectively review the code in question.
+- Rustdoc team members are members of the Rust organization so they can modify labels and be
+  assigned to issues.
+- Members become a part of the `rust-lang/rustdoc` team on GitHub, so that they receive pings
+  when people are looking to address the team as a whole.
+- Members are [listed] on the rust-lang.org web page.
+
+It also implies some obligations (in some cases, optional obligations):
+
+- Members may take part in various other [maintainer activities] to help the team.
+- Members are held to a higher standard than ordinary folk when it comes to the [Code of
+  Conduct][CoC].
+
+[listed]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
+
+## What it means to be a rustdoc member
+Once you're a member of the rustdoc team, a number of events will happen:
+
+- You will gain access to a private Zulip stream, where internal discussions happen.
+- You will also be subscribed to the `all@rust-lang.org` mailing list. See
+  [this file](https://github.com/rust-lang/team/blob/HEAD/teams/all.toml) to check how subscriptions
+  to mailing lists work. It's a very low-volume mailing list (maybe a few emails per year), it's a
+  way to communicate things to all contributors. We will not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+These groups are NOT full-fledge teams, and as such, to be part of any of these groups, you need to
+be a member of the rustdoc team.
+
+For now, only the front-end group has an official status in the
+[team repository](https://github.com/rust-lang/team). If a rustdoc team member is interested to be
+part of this group, they can ask to be added into it.
+
+Being part of the front-end group doesn't change your rustdoc team membership. However
+you will be assigned for reviews on front-end pull requests and on front-end FCPs.
+
+## How promotion decisions are made
```

Comment:
Apparently the Forge T-compiler don't mention this but for T-compiler it's mandatory (?) (or at least expected / *strongly* recommended) to do nominations *privately* (i.e., in the private Zulip channel), so objections can be raised more freely and in confidence (and in order to avoid sending false signals to the nominee in case the nomination gets rejected (e.g., on grounds of being too *premature*).

In my opinion, I think we should codify this here since I know from my limited experience that this a question that which did come up a non-zero amount of times. This would clarify it once and for all.

It needn't be a obligation if anyone is uncomfortable with it, it can simply remain a *strong recommendation* for now (while listing the aforementioned reasons).

Reply:
I didn't even pay attention. Yes I agree, it should be done privately.

Reply:
I precise: for now I put privately, but since we have to go through an FCP, it can discussed there. And even revisited in the future if someone sees the need to.